systeroid-tui/Cargo.toml

```diff
@@ -10,15 +10,15 @@ repository = "https://github.com/orhun/systeroid"
 keywords = ["linux", "kernel", "parameter", "sysctl", "tui"]
 categories = ["command-line-utilities"]
 edition = "2021"
-rust-version = "1.74.1"
+rust-version = "1.89.0"
```

Comment:
I think we need to update the `Dockerfile` accordingly, otherwise the builds will fail.

Reply:
Yes, this is true. I forgot to do that. I remember now that "old" images caused issues in the past.

Reply:
Done. Can you please start the workflow again?

Reply:
btw, most of the dependabot PRs can be closed. but maybe it's better to wait until this one is merged.